pkg/errors/errors.go

```diff
@@ -285,3 +288,11 @@ func IsClosedNetwork(err error) bool {
 func IsErrGitHubNotVerifiedEmail(err error) bool {
 	return err.Error() == ErrGitHubNotVerifiedEmail.Error()
 }
+
+func IsReadinessProbeFailed(err error) bool {
+	return err == ErrReadinessProbeFailed
```

Comment:
We should use `errors.Is()` instead of `==`: https://go.dev/wiki/ErrorValueFAQ#how-should-i-change-my-error-handling-code-to-work-with-the-new-features

Same for `IsLivenessProbeFailed`

Reply:
i've used directly error.Is and remove the custom function 👍🏻